editor/import/3d/post_import_plugin_skeleton_rest_fixer.cpp

```diff
@@ -41,8 +41,9 @@ void PostImportPluginSkeletonRestFixer::get_internal_import_options(InternalImpo
 	if (p_category == INTERNAL_IMPORT_CATEGORY_SKELETON_3D_NODE) {
 		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/apply_node_transforms"), true));
 		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/normalize_position_tracks"), true));
-		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/overwrite_axis"), true));
 		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/reset_all_bone_poses_after_import"), true));
+		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/overwrite_axis"), true));
+		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/skip_unmapped_ends"), false));
```

Comment:
```suggestion
		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/overwrite_axis", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_UPDATE_ALL_IF_MODIFIED), true));
		r_options->push_back(ResourceImporter::ImportOption(PropertyInfo(Variant::BOOL, "retarget/rest_fixer/preserve_leaf_axis_visually"), false));
```

As @lyuma's sugesstion `Preserve Leaf Axes`, so I suggest `Preserve Leaf Axis Visually` for consistency and for the sake of correctness of explanation of behavior.

Also needs `get_internal_option_visibility` implementation to hide it when `!bool(p_options["retarget/rest_fixer/overwrite_axis"])`, I have sent a PR #91641 that will be helpful as the reference for this, please check it out.

Reply:
The word "Visually" is confusing. It has no visual effect on imported models. just like "overwrite bone axis": it only affects the axis convention.

Just like Overwrite Axis, it affects which bones have overwritten axis.

The reason it appeared to have a visual effect in the original issue is because they had created content assuming a different convention, and the convention change led to different orientations.

Reply:
@lyuma So `Keep Global Rest on Leaf`? By “visually” I mean the global rest. The problem is that the value of local rest is changed even though it is set to hold.

Reply:
Aren't leaf bones dummies for determining the orientation or length of the last bone in a chain, like another bone after the fingertip bone? I don't think it fits the context here.

Reply:
Well, indeed. We originally should wanted to mean outside of Mapped End, but I don't think there is a good word for that.

Reply:
How about `Keep Global Rest for Leftovers`

Reply:
It'll be hard being descriptive with short names. There is an option as long as `Reset All Bone Poses After Import` so `Keep Global Rest on Leftovers` sounds good. It describes what the implementation is doing and in plain terms.